packages/manager/apps/telecom/src/app/telecom/pack/xdsl/meetings/xdsl-meetings.controller.js

```diff
@@ -0,0 +1,148 @@
+import get from 'lodash/get';
+import 'moment';
```

Comment:
`moment` is imported in the telecom application by using [`script-loader`](https://v4.webpack.js.org/loaders/script-loader).
It means that there is a global object available.
In addition, an [AngularJS constant](https://github.com/ovh/manager/blob/master/packages/manager/apps/telecom/src/components/moment/moment.constant.js) is also available (if needed).

Using an import statement from the `pack` section is not necessary IMO.
This can be done when this section will be extracted from the monolithic telecom app I guess.

Could you please validate that removing this import doesn't break anything?

Thank you!